Detector/DetSensitive/src/SimpleCalorimeterSD.cpp

```diff
@@ -15,9 +15,10 @@
 
 namespace det {
 SimpleCalorimeterSD::SimpleCalorimeterSD(const std::string& aDetectorName,
-  const std::string& aReadoutName,
-  const DD4hep::Geometry::Segmentation& aSeg)
-  : G4VSensitiveDetector(aDetectorName), m_calorimeterCollection(nullptr), m_seg(aSeg) {
+					 const std::string& aReadoutName,
+					 const DD4hep::Geometry::Segmentation& aSeg,
+					 bool aBirksLaw))
```

Comment:
This looks like a syntax error. Did you compile and test before opening the PR? (Also the bool argument is missing in the header.)